charmhelpers/core/host_factory/ubuntu.py

```diff
@@ -25,6 +25,7 @@
     'cosmic',
     'disco',
     'eoan',
+    'focal'
 )
```

Comment:
We should consider changing `UBUNTU_RELEASES` to a set

Reply:
i think the intent is that the order matters... but I see your point.  Maybe an 'OrderedSet' if there was such a thing...

Reply:
The order is very important; this particular #def is used to determine the order of releases and determine that "focal" comes after zesty, for example.